src/Umbraco.Cms.Api.Management/DependencyInjection/RelationTypesBuilderExtensions.cs

```diff
@@ -14,7 +14,6 @@ internal static IUmbracoBuilder AddRelationTypes(this IUmbracoBuilder builder)
             .Add<RelationTypeViewModelsMapDefinition>();
 
         builder.Services.AddTransient<IObjectTypePresentationFactory, ObjectTypePresentationFactory>();
-        builder.Services.AddTransient<IRelationTypePresentationFactory, RelationTypePresentationFactory>();
```

Comment:
I think removing this will clash with https://github.com/umbraco/Umbraco-CMS/pull/15933

Reply:
will check

Reply:
Yup, will do a forward merge when #15933 has been merged in dev and fix it